src/app.js

```diff
@@ -61,17 +62,17 @@ const options = {
 };
 app.use('/api-docs', swaggerUI.serve, swaggerUI.setup(swagger, false, options));
 
-
 passport.deserializeUser((id, done) => {
   User.findByPk(id)
     .then((user) => done(null, user))
     .catch((err) => done(err, null));
 });
 
 // Routes
-app.use("/auth", otpAuthRouter);
+app.use('/api/product/registration', prodRoute);
```

Comment:
this route should be improved >>/api/product/registration>>

Reply:
Done!